mne/time_frequency/spectrum.py

```diff
@@ -1192,6 +1232,10 @@ class SpectrumArray(Spectrum):
         The power spectral density for each channel.
     %(info_not_none)s
     %(freqs_tfr)s
+    %(method_psd)s
+    weights : ndarray
+        The weights used for averaging across tapers. Only required
+        for :func:`mne.time_frequency.psd_array_multitaper` with ``output='complex'``.
```

Comment:
per my most recent comment, let's assume YAGNI on supporting complex data with `(Epochs)SpectrumArray`, so this param can go away.